commit 68215d47cc7620345fb3439477eaac019ae42ab5 (refs/changes/68/4568/1)
Author: Rui Loura <rui@joyent.com>
Date:   2018-07-25T18:21:45+00:00 (1 year, 2 months ago)
    
    TRITON-603 cnapi should be aware of alternate admin nictags (bump version)

diff --git a/package.json b/package.json
index 6b55537..7b04baa 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cnapi",
   "description": "SmartDataCenter Compute Node API",
-  "version": "1.10.0",
+  "version": "1.11.0",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
